app/DoctrineMigrations/Version208.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * CAdding general management costs to report table
+ */
+class Version208 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('ALTER TABLE report ADD prof_dc_estimate_management_cost DOUBLE PRECISION DEFAULT NULL');
```

Comment:
For monetary amounts I believe we usually store as `NUMERIC(14, 2)` which is recommended for monetary fields.